lib/ferrum_pdf/railtie.rb

```diff
@@ -5,5 +5,13 @@ class Railtie < ::Rails::Railtie
         include FerrumPdf::Controller if FerrumPdf.include_controller_module
       end
     end
+
+    initializer "ferrum_pdf.load_configuration" do
+      config.after_initialize do
+        FerrumPdf.configure do |config|
+          # Load default configuration here if needed
+        end
+      end
+    end
```

Comment:
I don't think this does anything, so we should remove it.

Reply:
yes. it's doing nothing.